pkg/karmadactl/interpret/interpret.go

```diff
@@ -0,0 +1,168 @@
+package interpret
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/resource"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+	"github.com/karmada-io/karmada/pkg/karmadactl/options"
+	"github.com/karmada-io/karmada/pkg/karmadactl/util"
+	"github.com/karmada-io/karmada/pkg/util/gclient"
+)
+
+var (
+	interpretLong = templates.LongDesc(`
+        Check the valid of customizations. Or show the result of the rules running.
+
+        1. Check valid. Given the customization by NAME or '-f' option, this tool will
```

Comment:
I think the `-f` is enough, `NAME` here is kind of confusing and redundant. 

I know there are similar examples in kubectl, like `describe`, the difference is here we will accept arbitrary file names, but `describe` only accepts a certain amount of resource types. For instance, 
- kubectl describe pods   // all people know it will describe a `Pods` resource, because it's a well-known name.
- karmadactl interpreter example  // what's the `example` thing?

Reply:
updated